docker/mariadb/00_database-create-geokrety-db.sql

```diff
@@ -1,216 +1,155 @@
   `userid` bigint(20) NOT NULL,
   `user` varchar(30) COLLATE utf8_polish_ci NOT NULL,
-  `remember` binary(1) NOT NULL DEFAULT '0'
+  `remember` binary(1) NOT NULL DEFAULT '0',
+  KEY `userid` (`userid`)
 ) ENGINE=MyISAM DEFAULT CHARSET=utf8 COLLATE=utf8_polish_ci;
 
--- --------------------------------------------------------
-
---
--- Table structure for table `gk-badges`
---
 
 CREATE TABLE `gk-badges` (
-  `id` int(11) NOT NULL,
+  `id` int(11) NOT NULL AUTO_INCREMENT,
   `userid` bigint(20) NOT NULL,
   `timestamp` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
   `desc` varchar(128) COLLATE utf8_polish_ci NOT NULL,
-  `file` varchar(32) COLLATE utf8_polish_ci NOT NULL
+  `file` varchar(32) COLLATE utf8_polish_ci NOT NULL,
+  PRIMARY KEY (`id`),
+  KEY `timestamp` (`timestamp`),
+  KEY `userid` (`userid`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_polish_ci COMMENT='badges for the users';
 
--- --------------------------------------------------------
-
---
--- Table structure for table `gk-errory`
---
 
 CREATE TABLE `gk-errory` (
-  `id` int(10) UNSIGNED NOT NULL,
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
   `uid` varchar(50) COLLATE utf8_polish_ci NOT NULL DEFAULT '',
-  `userid` int(10) UNSIGNED NOT NULL,
-  `ip` varchar(16) COLLATE utf8_polish_ci NOT NULL,
+  `userid` int(10) unsigned NOT NULL,
+  `ip` varchar(46) COLLATE utf8_polish_ci NOT NULL,
   `file` text COLLATE utf8_polish_ci NOT NULL,
   `details` mediumtext COLLATE utf8_polish_ci NOT NULL,
   `severity` int(10) NOT NULL DEFAULT '0',
   `date` datetime NOT NULL,
-  `timestamp` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP
-) ENGINE=MyISAM DEFAULT CHARSET=utf8 COLLATE=utf8_polish_ci COMMENT='GK errors';
-
--- --------------------------------------------------------
+  `timestamp` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+  PRIMARY KEY (`id`),
+  KEY `date` (`date`),
+  KEY `severity` (`severity`),
+  KEY `userid` (`userid`)
+) ENGINE=MyISAM DEFAULT CHARSET=utf8 COLLATE=utf8_polish_ci;
 
---
--- Table structure for table `gk-geokrety`
---
 
 CREATE TABLE `gk-geokrety` (
-  `id` int(10) UNSIGNED NOT NULL,
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
   `nr` varchar(9) COLLATE utf8_polish_ci NOT NULL,
   `nazwa` varchar(75) COLLATE utf8_polish_ci DEFAULT NULL,
   `opis` text COLLATE utf8_polish_ci,
-  `owner` int(10) UNSIGNED DEFAULT NULL,
+  `owner` int(10) unsigned DEFAULT NULL,
   `data` datetime DEFAULT NULL,
-  `droga` int(10) UNSIGNED NOT NULL DEFAULT '0' COMMENT 'road traveled in km',
-  `skrzynki` smallint(5) UNSIGNED NOT NULL DEFAULT '0' COMMENT 'number of visited cache box',
-  `zdjecia` smallint(5) UNSIGNED NOT NULL DEFAULT '0',
-  `ost_pozycja_id` int(10) UNSIGNED NOT NULL DEFAULT '0' COMMENT 'position last ruchy id',
-  `ost_log_id` int(10) UNSIGNED NOT NULL DEFAULT '0' COMMENT 'log last ruchy id',
+  `droga` int(10) unsigned NOT NULL,
```

Comment:
Here also about comments, it was kinda nice and understandable...

Reply:
fixed